src/Field.ts

```diff
@@ -74,8 +75,8 @@ const setupFieldProps = (instance: FieldInterface, props: any, data: any) =>
     $inputMode: applyFieldPropFunc(instance, props.$inputMode || data?.inputMode || undefined),
     $validators: applyFieldPropFunc(instance, props.$validators || data?.validators || null),
     // other props
-    $hooks: props.$hooks || data?.hooks || {},
-    $handlers: props.$handlers || data?.handlers || {},
+    $hooks: observable(props.$hooks || data?.hooks || {}),
```

Comment:
I think we can move this `observable` into `makeObservable` as annotation:
makeObservable(this, {
...
$hooks: observable,
$handlers: observable,
})